packages/components/package.json

```diff
@@ -62,17 +62,17 @@
     "formsy-react-components": "~1.1.0",
     "history": "~4.7.2",
     "immer": "~10.0.2",
-    "immutable": "~3.8.2",
+    "immutable": "^3.8.2",
```

Comment:
nit: Did you mean to change these dependency declarations?

Reply:
This issue is because I used NPM to bump api-js, and it saw that the peer deps use `^` and our actual deps use `~`, so I updated our package.json to use `~` in our peer deps.

Reply:
I reverted the changes, it caused other issues downstream that I'll resolve later.